src/main/java/com/sollace/coppergolem/OxidizableCopperButtonBlock.java

```diff
@@ -13,6 +13,7 @@ protected OxidizableCopperButtonBlock(Oxidizable.OxidationLevel oxidizationLevel
         super(oxidizationLevel, settings);
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
The changes involved in fixing this are rather invasive (as far as I can tell), so I left it out for now.